app/modules/integrations/integrations_service.py

```
@@ -1715,7 +1654,7 @@ async def save_linear_integration(
             }
 
         except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)